charts/litmus/values.yaml

```
@@ -12,16 +12,21 @@ customLabels:
 # my.company.com/concourse-cd: 2
 
 adminConfig:
-  DBUSER: "admin"
-  DBPASSWORD: "1234"
+  # -- use already present secret
+  existingSecret: ""
   JWTSecret: "litmus-portal@123"
   VERSION: "2.9.0"
-  SKIP_SSL_VERIFY: "false"
+  SKIP_SSL_VERIFY: true
   # -- leave empty if uses Mongo DB deployed by this chart
+  DBPASSWORD: ""
+  DBUSER: ""
```

These are not used anymore

---

these are still used in case of specifying an external DB

---

```suggestion
  # -- leave empty if uses Mongo DB deployed by this chart
  DB_PASSWORD: ""
  DB_USER: ""
```

make these variables naming coherent